src/components/CaptureMatching/CaptureImage.js

```diff
@@ -128,30 +134,30 @@ function CaptureImage(props) {
                       Capture {(capture.id + '').substring(0, 10) + '...'}
                     </Typography>
                   </Tooltip>
-                  <Box className={classes.box3}>
-                    <AccessTimeIcon />
-                    <Typography variant="body1">
-                      {getDateTimeStringLocale(capture.created_at)}
-                    </Typography>
-                  </Box>
-
-                  <Box className={classes.box3}>
-                    <LocationOnOutlinedIcon />
-                    <Typography variant="body1">
-                      <Country lat={capture.lat} lon={capture.lon} />
-                    </Typography>
+                  <Box className={classes.captureInfo}>
+                    <Box className={classes.box3}>
+                      <AccessTimeIcon />
+                      <Typography variant="body1">
+                        {getDateTimeStringLocale(capture.created_at)}
+                      </Typography>
+                    </Box>
+                    <Box className={classes.box3}>
+                      <LocationOnOutlinedIcon />
+                      <Typography variant="body1">
+                        <Country lat={capture.lat} lon={capture.lon} />
+                      </Typography>
+                    </Box>
                   </Box>
-                  {/* <UseLocation/> */}
                 </Box>
 
                 <Grower
-                  planter_photo_url={capture.planter_photo_url}
-                  planter_username={capture.planter_username}
-                  status={capture.status}
+                  grower_photo_url={capture.grower_photo_url}
```

Comment:
Looks like I jumped the gun on this one.
The API is being changed to only include `grower_account_id` under Greenstand/treetracker-api#55.
I guess we'll need to look up the grower info instead.

Reply:
I think we should keep this until the API changes.